src/compas_timber/elements/beam.py

```diff
@@ -467,7 +485,28 @@ def remove_blank_extension(self, joint_key=None):
         else:
             del self._blank_extensions[joint_key]
 
+    def side_as_surface(self, side_index):
+        # type: (int) -> compas.geometry.PlanarSurface
+        """Returns the requested side of the beam as a parametric planar surface.
+
+        Parameters
+        ----------
+        side_index : int
+            The index of the reference side to be returned. 0 to 5.
+
+        """
+        # TODO: maybe this should be the default representation of the ref sides?
```

Comment:
I believe so as well. Using a parametric planar surface for the ref_side makes it more comprehensible i think.

Reply:
alright then I'd keep a mental note of maybe removing the frame-like ref sides. and we get rid of them in the future

Reply:
this comment was clarified in another PR, you can ignore my comment